src/components/Avatar/Avatar.styles.ts

```diff
@@ -90,21 +85,30 @@ const getAvatarSizeCss = (size: AvatarSize): SerializedStyles => {
       return previewAvatarCss
     case 'cover':
       return coverAvatarCss
-    case 'view':
-      return viewAvatarCss
     case 'channel':
       return channelAvatarCss
     case 'channel-card':
       return channelCardAvatarCss
     case 'fill':
       return fillAvatarCss
+    case 'bid':
+      return bidAvatarCss
     case 'small':
       return smallAvatarCss
     default:
       return defaultAvatarCss
   }
 }
 
+export const Outline = styled.div`
+  ${square('100%')};
+
+  border: 1px solid ${cVar('colorBorderAlpha')};
+  border-radius: 50%;
+  z-index: ${zIndex.overlay};
+  pointer-events: none;
+`
```

Comment:
Why do we need a separate element for that? Couldn't we just use `box-shadow` directly on the `Container`?